doc/FAQ

```diff
@@ -0,0 +1,101 @@
+- NodeOS - Lightweight operating system using Node.js as userspace - FAQ -
```

Comment:
Good idea to put the FAQ in the `doc` folder, maybe we should move all the wiki pages here and clean them too? There was some work to move them to an independent GitBook project...